regression-test/conf/regression-conf.groovy

```diff
@@ -60,3 +60,8 @@ hdfsFs = "hdfs://127.0.0.1:9000"
 hdfsUser = "doris-test"
 hdfsPasswd = ""
 brokerName = "broker_name"
+
+// broker load test config
+enableBrokerLoad=false
+awsAccessKey=""
+awsSecretKey=""
```

Comment:
There are already ak sk config item.

Reply:
Where can I find these config items? I didn't find it.